modules/navier_stokes/doc/content/syntax/Modules/NavierStokes/index.md

```diff
@@ -1 +1,15 @@
-!template load file=stubs/moose_system.md.template name=NavierStokes syntax=/Modules/NavierStokes
\ No newline at end of file
+# NavierStokes System
+
+## Overview
+
+!! Replace this line with information regarding the NavierStokes system.
+
+## Example Input File Syntax
+
+!! Describe and include an example of how to use the NavierStokes system.
+
+!syntax list /Modules/NavierStokes objects=True actions=False subsystems=False
+
+!syntax list /Modules/NavierStokes objects=False actions=False subsystems=True
+
+!syntax list /Modules/NavierStokes objects=False actions=True subsystems=False
```

Comment:
Can you remove:
```
- /Modules/NavierStokes
```
from `modules/navier_stokes/doc/hidden.yml`? Sinse you added doco, we need to remove it from the hidden file.

Reply:
I am not quite familiar with MooseDocs, will take a more careful look.

Reply:
we made the doco target/builds more draconian, so it will force you improve your moosedocs skills 🙃